test/shopify-cli/core/monorail_test.rb

```diff
@@ -182,6 +182,7 @@ def test_log_send_event_returns_result_if_timeout_occurs
       private
 
       def enabled_and_consented(enabled, consented)
+        ShopifyCli::Config.stubs(:get_section).with('analytics').returns({ 'enabled' => consented.to_s })
```

Comment:
The new check highlighted that were missing a stub here. Without this, some tests were hitting `ShopifyCli::Config.set` and writing to the config file.